base/range.jl

```diff
@@ -28,14 +28,40 @@ _colon(::Ordered, ::ArithmeticRounds, start::T, step, stop::T) where {T} =
 _colon(::Any, ::Any, start::T, step, stop::T) where {T} =
     StepRangeLen(start, step, floor(Integer, (stop-start)/step)+1)
 
+"""
+	(:)(expr)
+
+`:expr` quotes `expr`, returning an abstract syntax tree (AST) of `expr`.
+The AST may be of type `Expr`, `Symbol`, or a literal value.
+Which of these three types are returned for any given expression is an
```

Comment:
Does this sentence add anything useful?

Reply:
I get what you mean, but part of what I found confusing in the original issue was what type was actually returned.